python/wrappers.py

```diff
@@ -0,0 +1,31 @@
+def runCommand(cmd,log=None):
```

Comment:
You are not the original author, cite original location and author at the very least.